TombEngine/Game/Setup.h

```diff
@@ -142,7 +142,49 @@ struct ObjectInfo
 	}
 };
 
-struct STATIC_INFO
+class ObjectHandler
+{
+	private:
+		ObjectInfo m_objects[ID_NUMBER_OBJECTS];
```

Comment:
TEN conventions use PascalCase for member variables. The renderer and scripting system are currently exceptions since they were created before any hard rules were set.